spec/acceptance/ios_aaa_authentication_spec.rb

```diff
@@ -1,6 +1,36 @@
 require 'spec_helper_acceptance'
 
 describe 'ios_aaa_authentication' do
+  before(:all) do
```

Comment:
These entries should not be here. This is for setting up dependencies for the tests.